src/createDynamicHostResolver.js

```diff
@@ -12,33 +12,60 @@
 
 var window = require('@adobe/reactor-window');
 
-module.exports = function (turbineEmbedCode, cdnAllowList, debugController) {
+module.exports = function (
+  turbineEmbedCode,
+  dynamicCdnEnabled,
+  cdnAllowList,
+  debugController
+) {
+  // A missing list means that we are not trying to dynamic replace (archives,
+  // sftp, no premium CDN option enabled on the company).
   // even an empty list is flagging to us that we're trying to enforce dynamic
-  var isDynamicEnforced = Array.isArray(cdnAllowList);
+  var isDynamicEnforced = Boolean(
```

Comment:
Why do you need this extra flag? Why checking cdnAllowList is not enough?

Reply:
This flag is being added to the library output to guard against someone in the future emitting empty arrays for `cdnAllowList` on accident

Reply:
I am not sure I understand. How can someone emit an empty array for cdnAllowList? And who is that someone?